docs/casa/plugins/accts-linking/index.md

```diff
@@ -0,0 +1,99 @@
+# Accounts Linking Plugin
+
+## Overview
+
+This plugin allows users to "link" their local Jans account with existing accounts at third-party identity providers like OIDC OPs and social sites, e.g. Apple, Facebook, Google, etc.
+
+Besides the usual onboarding of a plugin jar file in Casa, administrators must deploy a number of additional components. This will be regarded later. By now let's summary key points of the accounts linking experience:
```

Comment:
However, lets summarize the key points of the ....

Reply:
updated